update.sh

```diff
@@ -60,4 +61,5 @@ if [ -n "$(ls -A patches/*.patch 2>/dev/null)" ]; then
 		echo "Applying patch ${i}"
 		(cd src && ${PATCH} -p3 < "${dir}/${i}")
 	done
+	${MV} src/rpki-client.8 src/rpki-client.8.in 
```

Comment:
I suggest to take out rpki-client.8 from the for loop and handle it specially.
Seems to be nicer to do this:

```
+# handle man page specially so that the path are adjusted by configure
+$ {CP} "${rpkiclient_src}/rpki-client.8" src/rpki-client.8.in
```

I will take care of this. Also will add the patch to my patch collection branch so it will remain around.

Reply:
Hmm. The patch needs to be applied to rpki-client.8 and then the move needs to happen. So that idea does not work.

On the other hand the replacement of the placeholders by ./configure is not correct. I end up with '${prefix}/etc/rpki' in the man page. I guess the in processing needs to be done during build time.

Reply:
Ah, good catch. I just tested it with specific options (rather with defaults), which lead for me indeed to the desired result; I'll update my patch suggestion later.

Reply:
Cool, in that case I will stop looking into this. It seems this needs a specific rule in src/Makefile.am to replace the placeholders. Why can't it ever be easy with automake...